oat-sa/extension-tao-testqti-previewer
# feat: add user-friendly message when previewer failed parsing xml

Ticket: https://oat-sa.atlassian.net/browse/AUT-1010

## What's Changed

- Add user-friendly message when previewer failed parsing xml
## Dependencies PRs

- https://github.com/oat-sa/tao-core-sdk-fe/pull/166


## How to test
- create items
- create test with items
- go to Items and delete one or more items
- try to Preview test
- check error message
